src/Features/Blockcore.Features.Consensus/Blockcore.Features.Consensus.csproj

```diff
@@ -17,14 +17,17 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="LevelDB.Standard" Version="2.1.6.1" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.11" />
     <PackageReference Include="Microsoft.FASTER" Version="2020.2.1.1" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\Blockcore\Blockcore.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.1.1" />
```

Comment:
Why do we need a github package?

Reply:
ok I see it's for creating debug info for debugging in to nuget packages

Reply:
Yes, it's just an upgrade.